sql/sql_class.cc

```diff
@@ -863,6 +863,8 @@ void THD::cleanup_connection(void) {
   sp_cache_clear(&sp_proc_cache);
   sp_cache_clear(&sp_func_cache);
 
+  m_connection_certificate = "";
```

Comment:
Does this need to be wrapped in #ifdef HAVE_OPENSSL?

Reply:
Probably the opposite: any existing `HAVE_OPENSSL` uses should be removed from patches. E.g. Now CMakeLists.txt contains
`ADD_DEFINITIONS(-DHAVE_OPENSSL)            # TODO: remove #ifdef from C++ code`

I guess it is assumed that WolfSSL is sufficiently similar to OpenSSL, and any code specific for it can be guarded with HAVE_WOLFSSL.